webapp/src/components/home/homeContainer.js

```diff
@@ -50,7 +51,7 @@ export default class HomeContainer extends React.Component {
       }
       });
 
-    fetch(webAppConfig.api.url+'/activity',myInit)
+    fetch(webAppConfig.api.url + '/activity?count=20',myInit)

```

Comment:
I don't think this completely resolves the issue. The fact that the panel is not responsive to the size of the viewport is the bigger issue that this needs to address.